cmd/process-agent/collector.go

```diff
@@ -137,7 +134,15 @@ func (l *Collector) run(exit chan bool) {
 					<-l.send
 				}
 				for _, m := range payload.messages {
-					l.postMessage(payload.endpoint, payload.name, m)
+					extraHeaders := map[string]string{
+						"X-Dd-Hostname":            l.cfg.HostName,
```

Comment:
Do these warrant being added as consts (separate file?) due to their importance?